From 2cc49155b15bab1a047fed6567e471308bc81c83 Mon Sep 17 00:00:00 2001
From: Kevin Tung <kevin.tung.openbmc@gmail.com>
Date: Tue, 18 Nov 2025 15:51:27 +0800
Subject: [PATCH] ARM: dts: aspeed: yosemite5: configure dimm ids for
 sbrmi node

Configure the DIMM count and DIMM ID mapping in the SBRMI node to
enable DIMM temperature and power monitoring.
This functionality is not supported in the upstream device tree,
so the change is applied only in the Meta tree.

Signed-off-by: Kevin Tung <kevin.tung.openbmc@gmail.com>
---
 arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite5.dts | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite5.dts b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite5.dts
index 5cafb6b7d..f119c82c4 100644
--- a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite5.dts
+++ b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite5.dts
@@ -219,6 +219,9 @@ &i2c3 {
 	sbrmi@3c {
 		compatible = "amd,sbrmi";
 		reg = <0x3c>;
+		dimm-count = <16>;
+		dimm-ids = <0x80 0x81 0x82 0x83 0x84 0x85 0x86 0x87
+				0x88 0x89 0x8a 0x8b 0x8c 0x8d 0x8e 0x8f>;
 	};
 
 	/* Socket 0 SBTSI */
-- 
2.47.3

